### PR TITLE
Pr karasz config

### PR DESCRIPTION
**Pull request by karasz**
add niceties, flag management and so on

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"os/signal"
+	"syscall"
+
+	"github.com/miekg/dns"
+	"github.com/spf13/viper"
+)
+
+// ListenAndServe creates a new Gnocco server and starts listening
+func ListenAndServe() error {
+	NewConfig()
```
Comment (amery) [inline server.go]: eh?

File: server.go [hunk 2]
```diff
@@ -7,15 +7,73 @@ import (
...
+	l.Print(msg)
+
+	panic("unreachable")
+}
+
+func init() {
+	log = zerolog.NewLogger(DefaultLogLevel)
+
+	// For now we only allow default config
+	cfgFile = DefaultConfigFile
```
Comment (amery) [inline server.go]: nasty

**Review by amery — CHANGES_REQUESTED** [review]
considering you add and then remove code the last two commits should probably be combined, otherwise it's poison to the history

**Comment by karasz** [comment]
Everything should be squashed

**Comment by karasz** [comment]
squashed the last two commits

File: server.go (hunk 2 above)
Comment (karasz) [reply server.go]: not anymore

**Review by amery — APPROVED** [review]

**Pull request merged**